src/Output/JsonEachRow.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimPod\ClickHouseClient\Output;
+
+use function Safe\json_decode;
+use function Safe\sprintf;
+use function str_replace;
+
+final class JsonEachRow implements Output
```

Comment:
zkus vymyslet lepší název, tady jsem fakt pár minut tápal a nechápal co to dělá (plus na to nemáš separe test, takže jsem to musel vyčíst odjinud ...)

Reply:
jj test musim pridat. ale nazev si vymejslet nebudu :D https://clickhouse.yandex/docs/en/interfaces/formats/#jsoneachrow

Reply:
hm ok ch dementi :D ok